react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -277,11 +277,13 @@ const meshMapLayer = {
     meshValueRange: [2782, 3513],
     propertyTexture: "kh_netmap_25_m_normalized_margin.png",
     propertyValueRange: [2782, 3513],
-    rotDeg: 0,
+    rotDeg: 0, // default rotate around bounds' upper left corner.
+    //rotPoint: [432205 + (439400 - 432205) / 2, 6475078 + (6481113 - 6475078) / 2],  // rotate around middle
```

Comment:
Do you need commented line ? Line 281.

Reply:
Yes I was thinking to have it there for easy referencing and testing.